flutter_frontend_server/lib/server.dart

```diff
@@ -169,3 +182,92 @@ Future<int> starter(
   frontend.listenAndCompile(compiler, input ?? stdin, options, completer);
   return completer.future;
 }
+
+/// A [RecursiveVisitor] that replaces [Object.toString] overrides with
```

Comment:
Should this code go in a separate file/library?

Reply:
I started doing that but it seemed to be overly complicating things to me - these classes are pretty short right now and the style in the engine and flutter repos tend to be to group related things together rather than splitting files.

That said I don't have a strong opinion about this one either way.

Reply:
It probably makes more sense to move the transformations to (a) separate library(ies) when the second one gets added. Let's keep an eye out for that, or leave a comment/TODO.